src/app/containers/NewStoryPromo/PromoOrganisms/TopStoriesPromo/index.styles.jsx

```diff
@@ -0,0 +1,45 @@
+import { getPica } from '#legacy/gel-foundations/src/typography';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+} from '#app/legacy/gel-foundations/src/spacings';
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+} from '#app/legacy/gel-foundations/src/breakpoints';
+import styled from '@emotion/styled';
+import Promo from '../../Promo';
+
+export const StyledPromoHeading = styled(Promo.Heading)`
+  display: inline-block;
+  ${({ script }) => script && getPica(script)}
+`;
+
+export const TopStoriesPromoWrapper = styled.section`
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    padding: 0 ${GEL_SPACING};
+  }
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    padding: 0 ${GEL_SPACING_DBL};
+  }
+`;
+
+export const StoryPromoList = styled(Promo.UnorderedList)`
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    display: flex;
+    gap: ${GEL_SPACING};
+  }
+`;
+
+export const FlexListItem = styled(Promo.ListItem)`
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    flex-grow: 1;
+    flex-basis: 0;
+    margin: 0;
+  }
+`;
```

Comment:
This was a little confusing with all of these living in `index.styles.jsx` I expected to see all of these being imported into `index.jsx` but it looks only `TopStoriesPromoWrapper` is, maybe the rest of the styles should live within the `TopStoriesMolecules` directory closer to where they're used?

Reply:
Yes, nice spot. I reiterated that component and forgot to move the styles to the correct location.